app/common/adapter/binary/ApiBinary.ts

```
@@ -29,7 +29,7 @@ export class ApiBinary extends AbstractBinary {
       `${this.config.cnpmcore.sourceRegistry}/-/binary`;
     let url = `${apiUrl}/${binaryName}${dir}`;
     if (lastData && lastData.lastSyncTime) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)